syntax = "proto3";
package menu;





/*
UpdateMenu返回
*/
message UpdateMenuResponse {
    /*
    菜单id
    */
    string id = 1 ;
}

/*
UpdateMenuApi返回
*/
message UpdateMenuResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    menu.UpdateMenuResponse data = 4 ;
}

